### ISSUE TITLE
Rename to ngx-bs3-modal maybe?

### ISSUE THREAD

**Issue by amitdahan** [dougludlow/ng2-bs3-modal#211]
This is more of a suggestion, maybe a rename is due since most packages are now using the new "ngx" prefix over "ng2" which implies Angular 2.X which is no longer the case?

**Comment by dougludlow** [comment]
@amitdahan, yes, thank you for the suggestion. I'll look at updating the name of the project/package soon.

**Comment by dougludlow** [comment]
`ngx-bs-modal` looks available (for now) and allows for support for bootstrap 4 possibly in the future.

**Comment by amitdahan** [comment]
Sounds great! :)
